api/src/main/java/org/openmrs/module/pihcore/config/PihConfigService.java

```diff
@@ -94,20 +92,6 @@ public InitializerConfig getInitializerConfig() {
         return c;
     }
 
-    public ConfigDomain getPihConceptsConfig() {
-        ConfigDomain pihConceptsDomain = new ConfigDomain("concepts");
-        Collection<File> mdsFiles = MetadataSharingSetup.loadMdsFiles();
-        if (mdsFiles != null) {
-            for (File f : mdsFiles) {
-                ConfigFile mdsFile = new ConfigFile();
-                mdsFile.setPath(f.getName());
-                mdsFile.setEnabled(true);
-                pihConceptsDomain.getConfigFiles().add(mdsFile);
-            }
-        }
-        return pihConceptsDomain;
-    }
```

Comment:
@mseaton  I wasn't 100% sure what this was used for, can we confirm it's not longer needed?